azavea/franklin
# Rewrite links when importing data

## Overview

This PR updates the import command to rewrite links when importing a collection or catalog.

### Checklist


### Notes

The issue says to write an integration test, but this might be better suited to its own issue.

### Testing Instructions

- Start the database server
- Open a shell for the database and drop anything that's already there:
```
./scripts/dbshell
truncate table collections;
truncate table collection_items;
```
- Import a dataset:
`bloop run application -- import --catalog-root s3://rasterfoundry-development-data-us-east-1/catalog.json --db-host localhost --api-host whoa`
- Start the server and inspect the links - the should be wrong (host is `whoa`) but that shows the import worked

Closes #32
